src/main/kotlin/racingcar/Position.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar
+
+data class Position(private val position: Int) {
+    init {
+        require(position >= 0)
+    }
+
+    fun increase() = copy(position = position + 1)
```

Comment:
++ 연산자를 오버로딩하면 어떨까요?

```suggestion
    operator fun inc() = copy(position = position + 1)
```

Reply:
오 좋네요! 👍 
75bed2edfc36b24408a6a32386aae3b4a6780958 에서 적용하였습니다.